### PR TITLE
Allow tapping on TabBar indicator to switch tabs

### LINKED ISSUE #11522: Tab tap target it to small
Tap target does not include the tab indicator (which iOS MDK does).

@goderbauer

### PR DESCRIPTION
**Pull request by goderbauer**
This increases the hit region of a tab by the area below it, which displays the indicator for the currently selected tab.

Fixes https://github.com/flutter/flutter/issues/11522

### CODE REVIEW

**Comment by Hixie** [comment]
cc @HansMuller

**Comment by eseidelGoogle** [comment]
Ping. :)

File: packages/flutter/test/material/tabs_test.dart [hunk 1]
```diff
@@ -1007,14 +1008,14 @@ void main() {
               actions: SemanticsAction.tap.index,
               flags: SemanticsFlags.isSelected.index,
               label: 'TAB #0\nTab 1 of 2',
-              rect: new Rect.fromLTRB(0.0, 0.0, 108.0, 46.0),
+              rect: new Rect.fromLTRB(0.0, 0.0, 108.0, 48.0),
```
Comment (HansMuller) [inline packages/flutter/test/material/tabs_test.dart]: We should probably include more of the constants from tabs.dart in material/constants.dart however for starters you could use kTextTabBarHeight here.

File: packages/flutter/test/material/tabs_test.dart [hunk 2]
```diff
@@ -1120,4 +1121,40 @@ void main() {
...
+      initialIndex: 0,
+    );
+
+    await tester.pumpWidget(
+      new Material(
+        child: new Column(
+          children: <Widget>[
+            new TabBar(
+              controller: controller,
+              indicatorWeight: 30.0,
```
Comment (HansMuller) [inline packages/flutter/test/material/tabs_test.dart]: NICE.

**Comment by HansMuller** [comment]
LGTM

**Pull request merged**